src/containers/diseasePage/diseasePageAssociationsTable.js

```diff
@@ -1,3 +1,6 @@
+/* disable-eslint */
```

Comment:
It might not be the best idea to disable eslint entirely. If there are troublesome lines, maybe consider `// eslint-disable-next-line` or only disable eslint only for relevant sections of the code? https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

Reply:
Disable all the things!...  this was just left over from debugging.